.vib/harbor-core/goss/harbor-core.yaml

```diff
@@ -0,0 +1,35 @@
+group:      
+  harbor:          
+    exists: true  
+user:      
+  harbor:          
+    exists: true
+file:
+  /etc/core:        
+    exists: true          
+    mode: "0775"          
+    owner: harbor          
+    filetype: directory
+  /data:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+  /etc/core/certificates:
+    exists: true
+    filetype: symlink
+  /etc/core/ca_download:
+    exists: true
+    filetype: symlink
+  /etc/core/token:
+    exists: true
+    filetype: symlink
+command:
+  # Ensure permissions for Internal TLS
+  check-permissions-system-certs:
+    exec: ls -l /etc/pki/tls/certs/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.trust.crt /etc/ssl/certs/ca-certificates.crt 2>/dev/null | grep ".*-rw-rw-r--"
+    exit-status: 0
+  # Ensure swagger binary doesn't exist
+  check-swagger-binary:
+    exec: ls /opt/bitnami/harbor-core/bin/swagger 2>/dev/null
+    exit-status: 2
```

Comment:
We can move this test so it uses de built-in `file` instead of running a command.